### PR TITLE
Add OOM killer check

### PR DESCRIPTION
**Pull request by krisreeves-stripe**
r? @sjung-stripe

### CODE REVIEW

File: checks.d/oom.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        last = None
+
+        try:
+            fh = open(instance.get('logfile'), 'rt')
+        except IOError, err:
+            level = AgentCheck.UNKNOWN
+            if err.errno == 2:
+                level = AgentCheck.WARNING
+            else:
+                level = AgentCheck.CRITICAL
```
Comment (sjung-stripe) [inline checks.d/oom.py]: style nit: might as well have CRITICAL instead of UNKNOWN and remove the else clause

File: checks.d/oom.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        kernlogRE = re.compile(instance.get('kernel_line_regex'))
+        killedRE = re.compile(instance.get('kill_message_regex', re.IGNORECASE))
+
+        last = None
+
+        try:
+            fh = open(instance.get('logfile'), 'rt')
+        except IOError, err:
+            level = AgentCheck.UNKNOWN
+            if err.errno == 2:
```
Comment (sjung-stripe) [inline checks.d/oom.py]: nitpick: use `errno.ENOENT` instead of magic number

**Comment by sjung-stripe** [comment]
lgtm

**Comment by sjung-stripe** [comment]
lgtm

**Comment by cory-stripe** [comment]
Looks great! Nice work @krisreeves-stripe!

**Pull request merged**